docs/support/nxos.rst

```diff
@@ -66,4 +66,30 @@ As a result, merges are **not atomic**.
 Diffs
 _____
 
-Diffs for merges are simply the lines in the merge candidate config.
+Diffs for merges are simply the lines in the merge candidate config. 
+[Netutils](https://netutils.readthedocs.io/en/latest/) is used for creating the merge diff between the candidate and running configurations. 
+One caveat of using netutils diff of configurations is that the diff is performed offline and not online in the device.
+
+Example assuming that the device config contains:
+
+```
```

Comment:
One more: the doc build is failing on this, it seems: please indent and optionally add here `.. code-block:: text` (for the RST format). Same below, likely: `.. code-block:: python`.

Reply:
Yeah, it was markdown mixed in, I converted it to reST now.